lib/filterx/func-unset-empties.c

```diff
@@ -68,6 +70,19 @@ _should_unset(FilterXFunctionUnsetEmpties *self, FilterXObject *obj)
       return len == 0;
     }
 
+  if (!str)
+    return FALSE;
+
+  guint num_targets = self->targets ? self->targets->len : 0;
+
+  for (guint i = 0; i < num_targets; i++)
+    {
+      const gchar *target = g_ptr_array_index(self->targets, i);
+      if ((self->ignorecase ? strcasecmp(str, target) : strcmp(str, target)) == 0)
```

Comment:
that's true. 
originally utf8 wasn't in the scope, since non of our other string related function supporting utf8 atm. supporting utf8 could be useful, indeed.